contracts/p1/BackingManager.sol

```diff
@@ -99,6 +99,84 @@ contract BackingManagerP1 is TradingP1, IBackingManager {
         }
     }
 
+    /// This will allow the callee to start a dust auction for a specific asset.
+    /// No other active trades must be open and current basket must be fully collateralized.
+    /// @param sellToken The asset we have a dust balance of
+    /// @custom:interaction RCEI
+    function startDustAuction(IERC20 sellToken, TradeKind kind) external notTradingPausedOrFrozen {
+        // @to reviewer: should we let the user set the TradeKind?
+
+        // == Refresh ==
+        // @to reviewer: Not sure what needs to be called/refreshed here
```

Comment:
If we buy the RToken, then we need both these lines. If we don't buy the RToken then we could get away with just refreshing the buy and sell asset.